### ISSUE TITLE
Skip H1 headers

### ISSUE THREAD

**Issue by romainmartinez** [jupyterlab/jupyterlab-toc#66]
It is common to start a notebook with a title, like:

```
# My notebook title

Some information

## My first "real" title

> code
```
Here, I don't want the title of the notebook to be numbered.

An interesting feature of the [ToC 2](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/toc2) extension was to skip the h1 headers in the automatic numbering.

Is it possible to do the same with jupyterlab-toc? If not, is it a feature that could be included in the road map?

Thank you.

**Comment by mhogg** [comment]
I second this request :+1:

**Comment by romainmartinez** [comment]
I found a little hack. Just avoid putting an `h1` tag when you don't want the numbering and replace it with a font tag.

My example would become:

```
<font size="+2">My notebook title</font>

Some information

# My first "real" title

> code
```

**Comment by YubinXie** [comment]
I agree! 

And I'd like to thank the developers to add index function to Jupyter toc!!!

**Comment by dturaev** [comment]
Agreed, excluding some headers from the table of contents would be a very useful feature.

**Comment by gh4ag** [comment]
Any updates?

**Comment by pierre-haessig** [comment]
Is it fine to say that the [Jupyterlab PR #9994](https://github.com/jupyterlab/jupyterlab/pull/9994) merged on 2021-06-22 effectively closes this issue? Or should we wait for a nice UI to set the newly introduced setting `numberingH1`?

**Comment by fcollonval** [comment]
Yes this is indeed close by the mentioned PR. This issue is not updated here anymore as the code for the table of content is now directly in JupyterLab core and does not rely on the code of this repository.

**Comment by pierre-haessig** [comment]
Ah yes. Since I didn't land on the project home page, I didn't see the "Archived" banner in the README. The merge with core JupyterLab is good news for me.
